ipaserver/install/kra.py

```diff
@@ -53,6 +55,14 @@ def install_check(api, replica_config, options):
                 "KRA is not installed on the master system. Please use "
                 "'ipa-kra-install' command to install the first instance.")
 
+    parser = RawConfigParser()
+    parser.read(paths.IPA_DEFAULT_CONF)
+    ca_host = parser.get('global', 'ca_host', fallback=None)
```

Comment:
You don't need to re-parse the configuration file. You can use if api.env.ca_host is not None...